### PR TITLE
Update to 2023-09

### PR DESCRIPTION
**Pull request by rodtell**

### CODE REVIEW

File: org.eclipse.Java.metainfo.xml [hunk 1]
```diff
@@ -31,13 +31,13 @@
   <developer_name>Eclipse Foundation</developer_name>
   <update_contact>akurtakov@gmail.com</update_contact>
   <releases>
-	<release version="4.28" date="2023-06-14">
+	<release version="4.29" date="2023-09-13">
       <url>https://www.eclipse.org/eclipse/news/4.28/</url>
```
Comment (akurtakov) [inline org.eclipse.Java.metainfo.xml:35]: Please correct the versions in the news urls too.

Comment (rodtell) [reply org.eclipse.Java.metainfo.xml]: Sorry about that. I think I got it right this time.

**Comment by akurtakov** [comment]
This PR looks good itself. The error seems to come from Eclipse shipping with signed natives in the JNA bundle which the tooling doesn't seem to handle well.

**Comment by akurtakov** [comment]
I've took the liberty to reopen it as you did good work. Just some extra work would have to be done.

**Pull request closed without merging**